PollBuddy-Server/frontend/src/pages/GroupPolls/GroupPolls.js

```diff
@@ -1,60 +1,133 @@
 import React, {Component} from "react";
 import {Link} from "react-router-dom";
 import "./GroupPolls.scss";
-import {MDBContainer, MDBIcon} from "mdbreact";
+import {MDBContainer} from "mdbreact";
 
 export default class GroupPolls extends Component {
   constructor(props) {//shouldn't this be dependent on the class???? thats why i included a constructor.
     super(props);
     //need to connect to backend probably here and then store data until it can be stored in state.
     //problem is there is no find in backend rn... frontend could do find but probably more resource intensive?
+    //TODO: get all this from a backend call
     this.state = {
+      isMember: false,
+      class: "1200 - Data Structures",
+      polls: [
+        {pollId: 1, label: "Big O Notation"},
+        {pollId: 2, label: "Basic C++ Syntax"},
+        {pollId: 3, label: "Pointers"},
+        {pollId: 4, label: "Vectors"},
+        {pollId: 5, label: "Linked Lists"},
+        {pollId: 6, label: "Sets"},
+        {pollId: 7, label: "Maps"}
+      ],
+      total_questions: 24,
+      avg_correct: 20,
+      member_correct: 22
       //need to put in groupID from backend
       //need to get other shit like pollIDs and their respective information...
     };
   }
 
   componentDidMount() {
-    this.props.updateTitle("Polls");
+    this.props.updateTitle(this.state.class);
   }
 
+  toggleTextBox(elementId, selector, text) {
+    if(document.getElementById(elementId).style.display === "block") {
+      document.getElementById(elementId).style.display = "none";
+      document.querySelector(selector).textContent = text;
+    } else {
+      document.getElementById(elementId).style.display = "block";
+      document.querySelector(selector).textContent = "Submit";
+    }
+  }
   render() {
     return (
       <MDBContainer className="page GroupPolls">
```

Comment:
can probably remove GroupPolls class and the associated css file since this new version doesn't use any of it